superduper/misc/special_dicts.py

```diff
@@ -408,6 +408,20 @@ def _display_component(obj, verbosity=1):
 
     console = Console()
 
+    def _handle_list(lst):
+        handled_list = []
+        for item in lst:
+            if isinstance(item, Leaf):
+                if len(str(item)) > 50:
```

Comment:
since you are already making changes to the file, lets make 50 value a const var.